src/components/Calendar/Calendar.tsx

```diff
@@ -136,10 +143,18 @@ const CalendarStyling = styled.div`
     }
 
     .fc .fc-day-other {
-        background-color: ${(props) => props.theme.main.background[1]};
+        > div {
+            background-color: ${(props) => props.theme.main.background[2]};
```

Comment:
should this be using background2 without the color being foreground2?
and should the color be moved to within this too?

Reply:
Make sense. Refactored